### PR TITLE
Dates are now stored in UTC

### PR DESCRIPTION
**Pull request by djaiss**

### CODE REVIEW

**Comment by djaiss** [comment]
@asbiin could you review this?

File: app/Http/Controllers/ContactsController.php [hunk 1]
```diff
@@ -160,7 +160,7 @@ public function show(Contact $contact)
             $query->orderBy('updated_at', 'desc');
         }]);
 
-        $contact->last_consulted_at = Carbon::now(auth()->user()->timezone);
+        $contact->last_consulted_at = Carbon::now();
```
Comment (asbiin) [inline app/Http/Controllers/ContactsController.php]: You could use the `ǹow()` helper

File: app/Models/Contact/Contact.php [hunk 2]
```diff
@@ -1509,9 +1509,9 @@ public function updateStayInTouchFrequency($frequency)
      * @param int $frequency
      * @param string $timezone
      */
-    public function setStayInTouchTriggerDate($frequency, $timezone)
+    public function setStayInTouchTriggerDate($frequency)
     {
-        $now = Carbon::now($timezone);
+        $now = Carbon::now();
```
Comment (asbiin) [inline app/Models/Contact/Contact.php]: You could use the `ǹow()` helper

File: database/migrations/2018_06_14_212502_change_default_name_order_user_table.php [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::table('users', function ($table) {
+            $table->string('name_order')->default('firstname_lastname_nickname')->change();
```
Comment (asbiin) [inline database/migrations/2018_06_14_212502_change_default_name_order_user_table.php]: This has nothing to do with the UTC timezone, right ?

Comment (djaiss) [reply database/migrations/2018_06_14_212502_change_default_name_order_user_table.php]: No indeed. It's a bug fix for something else. I should put it out and create a separate PR for this.

Comment (asbiin) [reply database/migrations/2018_06_14_212502_change_default_name_order_user_table.php]: OK I have moved it in #1585

File: app/Http/Controllers/ContactsController.php (hunk 1 above)
Comment (djaiss) [reply app/Http/Controllers/ContactsController.php]: Done.

File: app/Models/Contact/Contact.php (hunk 2 above)
Comment (djaiss) [reply app/Models/Contact/Contact.php]: Done.

**Pull request merged**